btc-lsp/src/BtcLsp/Psbt/PsbtOpener.hs

```diff
@@ -69,36 +70,41 @@ mapLeaseUtxosToPsbtUtxo lockedUtxos = do
 
 fundChanPsbt ::
   (Env m) =>
+  Lnd.PendingChannelId ->
   [PsbtUtxo] ->
   OnChainAddress 'Fund ->
   OnChainAddress 'Gain ->
   Money 'Lsp 'OnChain 'Gain ->
   ExceptT Failure m Lnd.Psbt
-fundChanPsbt userUtxos chanFundAddr changeAddr lspFee = do
+fundChanPsbt pcid userUtxos chanFundAddr changeAddr lspFee = do
   let userFundingAmt = sumAmt userUtxos - coerce lspFee
 
+  let pcidTag = "[pcid:" <> inspect pcid <> "] "
```

Comment:
Probably better just to use `katipAddContext` combinator over all expression, to reduce code duplication. Just make sure function is not recursive and context is not added in every interation. Example:

https://github.com/coingaming/src/blob/4e156928e0146112abf9e1dbe729b8c62af5ebf3/btc-lsp/test/TestAppM.hs#L375-L382